packages/plugin-eslint/src/lib/runner/transform.unit.test.ts

```diff
@@ -221,6 +238,25 @@ describe('lintResultsToAudits', () => {
           ],
         },
       },
+      {
+        slug: 'typescript-eslint-prefer-nullish-coalescing',
+        score: 0,
+        value: 1,
+        displayValue: '1 warning',
+        details: {
+          issues: [
+            {
+              message:
+                'This rule requires the strictNullChecks compiler option to be turned on to function correctly',
+              severity: 'warning',
+              source: {
+                file: 'src/app/test/strictNullChecks.ts',
+                position: { startColumn: 1 },
+              },
```

Comment:
This is the result I would expect:

```suggestion
              source: {
                file: 'src/app/test/strictNullChecks.ts',
              },
```